### PR TITLE
Support custom scroll view

### PR DESCRIPTION
**Pull request by daisukefuji**

### CODE REVIEW

File: library/src/main/java/com/jmedeisis/draglinearlayout/DragLinearLayout.java [hunk 1]
```diff
@@ -47,6 +47,42 @@
     private final float nominalDistanceScaled;
 
     /**
+     * Use with {@link com.jmedeisis.draglinearlayout.DragLinearLayout#setContainerScrollViewListener(com.jmedeisis.draglinearlayout.DragLinearLayout.ScrollViewListener)}
+     * to scroll a view which contains draggable view.
+     */
+    public interface ScrollViewListener {
```
Comment (dagezi) [inline library/src/main/java/com/jmedeisis/draglinearlayout/DragLinearLayout.java]: `Listener` sounds weird here. Listener sounds like something just listening for some events passively.
How about `Scrollable`?

**Comment by dagezi** [comment]
Thanks! Almost looks good other than the naming.

**Comment by daisukefuji** [comment]
Renamed. fa9ad48

**Comment by dagezi** [comment]
LGTM

**Pull request merged**